Tests/L0.ts

```diff
@@ -201,6 +201,81 @@ describe('General Suite', function () {
         done();
     })
 
+    it('Find unsupported satisfied capabilities', (done) => {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
+
+        var supportedCapabilities: string[] = ['AndroidSDK',
```

Comment:
Can we share this across tests? That way we make sure it doesn't get out of sync. And in this test we can just clone it and append 'Node'

Reply:
Sorry for the long wait, I was oof.